src/CMakeLists.txt

```diff
@@ -1429,6 +1429,7 @@ if(${WITH_RADOSGW})
   endif(HAVE_SSL)
 
   add_library(rgw_a STATIC ${rgw_a_srcs})
+  target_include_directories(rgw_a PUBLIC ${FCGI_INCLUDE_DIR})
```

Comment:
I just did a rebuild with that line changed from PUBLIC to PRIVATE, and it failed, so I think that one needs to remain as-is.